src/main/java/com/royalrangers/service/UserService.java

```diff
@@ -133,6 +133,10 @@ public Boolean isEmailExist(String email) {
         return (userRepository.findByEmail(email) != null);
     }
 
+    public Boolean isTempUserExist(Long userId) {
+        return (tempUserRepository.findByUserId(userId) != null);
```

Comment:
Let's try to use exists method here. Not 100% sure it can be used, but lets check. 
More info - [here](https://stackoverflow.com/a/43509367)